csunplugged/topics/migrations/0047_curriculumarea_colour.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-05-05 08:47
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('topics', '0046_glossarydefinition'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='curriculumarea',
+            name='colour',
```

Comment:
Ah!! This is a migration from another branch!! No idea how that ended up there! Will delete!